### PR TITLE
cheetah/panther/lynx: Initial official build

### PR DESCRIPTION
**Pull request by DarkJoker360**

### CODE REVIEW

File: cheetah.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/cheetah/10.x/recovery/",
+			"paypal": "https://www.paypal.com/paypalme/Martiriggiano",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_pantah",
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline cheetah.json:25]: missing 14.0 branch

File: lynx.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Montly",
+			"forum": "https://xdaforums.com/t/rom-14-lynx-crdroid-10-25-12-2023-lynx.4647462/",
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/lynx/10.x/recovery/",
+			"paypal": "https://www.paypal.me/vikasyaduvanshii",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_lynx",
```
Comment (gwolf2u) [inline lynx.json:23]: missing 14.0 branch

File: lynx.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline lynx.json:25]: missing 14.0 branch

File: panther.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/cheetah/10.x/recovery/",
+			"paypal": "https://www.paypal.com/paypalme/Martiriggiano",
+			"telegram": "https://t.me/pixel7crdroid",
+			"dt": "https://github.com/crdroidandroid/android_device_google_pantah",
+			"common-dt": "https://github.com/crdroidandroid/android_device_google_gs201",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_google_gs201"
```
Comment (gwolf2u) [inline panther.json:25]: missing 14.0 branch

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by gwolf2u — APPROVED** [review]

File: panther.json (hunk 4 above)
Comment (dominator-prime) [reply panther.json:25]: thanks for informing 
updated lynx to  14.0 branch 
updated kernel 201 to 14.0 branch 
other pantah  already contain 14.0 branch

**Pull request merged**